### PR TITLE
Add list of claims card

### PR DESCRIPTION
**Pull request by tintinthong**
https://www.loom.com/share/8e71226b03cf4f8893cf881b13058cdc?sid=f4f38654-ee7f-494b-a7cd-d9b1dc6eb038

This [PR](https://github.com/cardstack/boxel/pull/430) should go in first

### CODE REVIEW

**Comment by tintinthong** [comment]
looks like context is available in box component via @context. `field-component` is the main file

**Comment by tintinthong** [comment]
Interesting. I encountered an error when using relative link to the claim card

```
Encountered error rendering HTML for card: Reflect.getPrototypeOf called on non-object
```

File: packages/base/links-to-many-component.gts [hunk 1]
```diff
@@ -157,6 +157,12 @@ export function getLinksToManyComponent({
...
     };
   } else {
-    return getPluralViewComponent(arrayField, field, format, cardTypeFor);
+    return getPluralViewComponent(
+      arrayField,
+      field,
+      format,
+      cardTypeFor,
+      context
+    );
```
Comment (tintinthong) [inline packages/base/links-to-many-component.gts:166]: @habdelra @jurgenwerk this is the place where the context was not passed down. It probably is a thing with contains-many component tooo

Comment (jurgenwerk) [reply packages/base/links-to-many-component.gts:166]: Nice find!

File: packages/demo-cards/claim.gts [hunk 2]
```diff
@@ -290,6 +292,39 @@ class Isolated extends Component<typeof Claim> {
   }
 }
 
+class Embedded extends Component<typeof Claim> {
+  get cannotClickViewClaimButton() {
```
Comment (jurgenwerk) [inline packages/demo-cards/claim.gts]: Maybe `claimButtonDisabled` is a simpler name for this?

File: packages/demo-cards/claim-list.gts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import StringCard from 'https://cardstack.com/base/string';
+import { Claim } from './claim';
+
+export class ClaimList extends Card {
+  static displayName = 'List of Claims';
+  @field about = contains(StringCard);
+  @field claims = linksToMany(() => Claim);
+  @field title = contains(StringCard, {
+    computeVia: function (this: ClaimList) {
+      return this.about;
```
Comment (jurgenwerk) [inline packages/demo-cards/claim-list.gts]: How come `about` and `title` are the same? Should `about` be more something like a description?

File: packages/demo-cards/claim.gts (hunk 2 above)
Comment (tintinthong) [reply packages/demo-cards/claim.gts]: Resolved https://github.com/cardstack/boxel/pull/418/commits/e71162a3c1bcc0df94a3870ac235dc6b5829696d

File: packages/demo-cards/claim-list.gts (hunk 3 above)
Comment (tintinthong) [reply packages/demo-cards/claim-list.gts]: Resolved https://github.com/cardstack/boxel/pull/418/commits/06083643c95b5989741b62375b552a884b320692 and https://github.com/cardstack/boxel/pull/418/commits/06083643c95b5989741b62375b552a884b320692. I dont know a good name for title so I made it into displayName.

Comment (tintinthong) [reply packages/demo-cards/claim-list.gts]: Screenshot 2023-06-26 at 20 31 21

**Review by jurgenwerk — APPROVED** [review]

**Review by habdelra — APPROVED** [review]

**Comment by tintinthong** [comment]
@jurgenwerk I had to fix a bug with modifier code that expects optional to be defined. Fixed it here
 https://github.com/cardstack/boxel/pull/418/commits/9ebceddf807c0fee8febd16aa6231ec05893d061

**Pull request merged**